Week1/homework/app.js

```diff
@@ -1,11 +1,138 @@
 'use strict';
 
 {
-  const bookTitles = [
-    // Replace with your own book titles
-    'harry_potter_chamber_secrets',
+  // Q1.1
+
+  const books = [
```

Comment:
The previous name ("bookTitles") was better; strive to always go for more specific and meaningful naming!